README.md

```diff
@@ -27,6 +27,7 @@ JDK is not required because of embedded into archive version is used.
 | youtrack_user | User to run YouTrack service | `youtrack` |
 | youtrack_group | Group for YouTrack user | `{{ youtrack_user }}` |
 | youtrack_home_dir | Home directory for YouTrack user | `/home/{{ youtrack_user }}` |
+| youtrack_base_dir | Location of the symbolic link for the latest version of youtrack installed | `{{ youtrack_home_dir }}/youtrack` |
```

Comment:
It's not a directory, why not to name it like `youtrack_release_link`?

Reply:
Fair enough, will update the branch.